### PR TITLE
Show 3-word names in profile screens in Desktop.

### LINKED ISSUE #6154: 3 words name is not shown in Profile on Desktop
### Description

*Type*: Bug

*Summary*: see https://github.com/status-im/status-react/pull/6128 implemented on mobile. Same should be done for desktop but it is not working.

#### Expected behavior

- 3 words name is shown in my profile along with my custom name
- 3 words name is shown in someone's profile along with custom name
- 3 words name is shown in chats along with custom name

### Reproduction

- Open Status
- Open profile
- Open someone's profile
- Open public chat

### PR DESCRIPTION
**Pull request by mandrigin**
fixes #6154

### Steps to test:
- Open Status (Desktop)
- a. Go to your profile
- b. Open profile of any chat user

Expected:
- see 3-word names under the selected profile name

status: ready

### CODE REVIEW

**Review by vitvly — APPROVED** [review]

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs [hunk 1]
```diff
@@ -31,7 +32,10 @@
      [react/text {:style           styles/profile-user-name
                   :font           :medium
                   :number-of-lines 1}
-      name])])
+      name])
+   [react/text {:style           styles/profile-three-words
```
Comment (flexsurfer) [inline src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: can we add condition `(not= name (gfy/generate-gfy public-key))` to do not show 3w name twice

Comment (mandrigin) [reply src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: makes sense

Comment (mandrigin) [reply src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: @flexsurfer done!

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs [hunk 2]
```diff
@@ -31,7 +32,11 @@
      [react/text {:style           styles/profile-user-name
                   :font           :medium
                   :number-of-lines 1}
-      name])])
+      name])
+   (when-not (= name (gfy/generate-gfy public-key))
```
Comment (rasom) [inline src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: not a big thing, `(when (not=` might be a bit better, though i'm puzzled with clojure's code style:

https://github.com/bbatsov/clojure-style-guide#not-equal (also https://github.com/bbatsov/clojure-style-guide#when-not but we don't have `(when (not` here :D )

**Review by rasom — APPROVED** [review]

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs (hunk 2 above)
Comment (mandrigin) [reply src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: idk, should we add a PR to their repo to add this case?
I don't have any strong opinions on that one tho

**Comment by churik** [comment]
@mandrigin one question.
Why "Unknown" stops displaying when you add user to contacts from public chat?
You still cannot see real name until user doesn't add you as contact.
umnko

Video: http://take.ms/gkEn9

<blockquote><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/fDgrUym9UPr93KIdmDnjitHJ1GTtun">File "screencast 2018-10-09 17-29-29.mp4"</a></strong></div><div>Monosnap — the best tool to take & share your screenshots.</div></blockquote>

**Comment by mandrigin** [comment]
@churik hmm, looks like a bug... I'll fix it

**Comment by mandrigin** [comment]
@churik for the record, the bug is that we show "Unknown" at all, we shouldn't do it. So my fix is that it should never show "Unknown" again.

**Comment by churik** [comment]
Tested (on IOS and desktop): 
- user profile
- profiles of other users  when navigating from public channels and from 1-1 chats
- adding user to contacts when navigating to profile from public channel
- adding user to contacts when pasting contact code

**Pull request merged**